client_generic/ContentDownloader/SheepGenerator.cpp

```diff
@@ -302,6 +302,13 @@ bool	SheepGenerator::getControlPoints( SheepUploader *uploader )
 																					1.0,
 																					1.0 );
 
+	// Convert "r3d0.sheepserver.net" to "v3d0.sheepserver.net" 
```

Comment:
I tend to think that any try to fix server bug on client side is not good. It should be temporary solution. I agree that it could be done in more elegant way.
For example,
1. 
#define SERVER_REPLACING
2. 
#define SERVER_VOTING_REGISTERED_USERS    "r3d0.sheepserver.net"
#define SERVER_VOTING_UNREGISTERED_USERS   "v3d0.sheepserver.net"
3. 
#ifdef SERVER_REPLACING
   char *pChr = std::strstr(url, SERVER_VOTING_REGISTERED_USERS);

   if (NULL != pChr)
   {
    // replacing the rendering server of registered users  to the server of unregistered users
    char url_tmp[MAX_PATH];
    *pChr = 0;
    snprintf(url_tmp, MAX_PATH, "%s%s%s", url, SERVER_VOTING_UNREGISTERED_USERS, &pChr[std::strlen(SERVER_VOTING_REGISTERED_USERS)]);
    strcpy(url, url_tmp);
   }
#endif